src/GitReportCL/DateAndPathValidation.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO;
+namespace GitReport.CLI
+{
+    class DateAndPathValidation                        
+    {
+        public bool ValidateGitDiffReportArgs(string[] arguments, GitReportArguments getArg)         
```

Comment:
Methods which returnes boolean value should indicate that in method name.
For example: `IsValid`, `HasSomething` etc.